dbtmetabase/parsers/dbt_folder.py

```diff
@@ -0,0 +1,223 @@
+            # We are implicitly complying with aliases by doing this
+            name=model.get("identifier", model["name"]).upper(),
+            schema=schema,
+            description=description,
+            columns=mb_columns,
+        )
+
+    def _read_column(self, column: Mapping, schema: str) -> MetabaseColumn:
+        """Reads one dbt column in Metabase-friendly format.
+
+        Arguments:
+            column {dict} -- One dbt column to read.
+
+        Returns:
+            dict -- One dbt column in Metabase-friendly format.
+        """
+
+        mb_column = MetabaseColumn(
+            name=column.get("name", "").upper().strip('"'),
+            description=column.get("description", ""),
+        )
+
+        for test in column.get("tests", []):
+            if isinstance(test, dict):
+                if "relationships" in test:
+                    relationships = test["relationships"]
+                    mb_column.semantic_type = "type/FK"
+                    # Note: For foreign keys that point to a different schema than the target, the yml meta: metabase.fk_ref must be used
+                    # Otherwise we use target schema which should be fine in 95% of cases
+                    mb_column.fk_target_table = (
+                        column.get("meta", {})
+                        .get(
+                            # Prioritize explicitly set FK in yml file which should have format schema.table unaliased
+                            "metabase.fk_ref",
+                            # If metabase.fk_ref not present in YAML, infer FK relation to table in target schema and parse ref/source
+                            # We will be translating any potentially aliased source() calls later during FK parsing since we do not have all possible aliases yet and thus cannot unalias
+                            self.parse_ref(relationships["to"], schema),
+                        )
+                        .strip('"')
+                    )
+                    if not mb_column.fk_target_table:
+                        logging.warning(
+                            "Could not resolve foreign key target for column %s",
+                            mb_column.name,
+                        )
+                        continue
+                    # Lets be lenient and try to infer target schema if it was not provided when specified in metabase.fk_ref
+                    # Because parse_ref guarantees schema.table format, we can assume this was derived through fk_ref
+                    if not "." in mb_column.fk_target_table:
+                        logging.warning(
+                            "Target table %s has fk ref declared through metabase.fk_ref missing schema (Format should be schema.table), inferring from target",
+                            mb_column.fk_target_table,
+                        )
+                        mb_column.fk_target_table = (
+                            schema + "." + mb_column.fk_target_table
```

Comment:
Use f-strings, they're more readable.

Reply:
f strings used for readability improvement